app/partials/content.html

```diff
@@ -1,15 +1,68 @@
-<div class="row marketing">
-  <div class="col-lg-6">
-    <h4>HTML5 Boilerplate</h4>
-    <p>HTML5 Boilerplate is a professional front-end template for building fast, robust, and adaptable web apps or sites.</p>
-
-    <h4>Sass</h4>
-    <p>Sass is the most mature, stable, and powerful professional grade CSS extension language in the world.</p>
+<div class="row products uppercase">
+    <div class="col-lg-2 col-md-offset-1">
+      <img src="images/prod1.png"/></div>
+    <div class="col-lg-1">
+      <p>branded</p>
+      <p>shoes</p>
+      <button type="button" class="btn btn-success uppercase">shop</button>
```

Comment:
I think this branded shoows, should be a heading, like an h2 or something similar, and not a paragraph, because it doesn't have a meaning of a paragraph.
Also, by splitting it in 2 elements, you're thinking in design mode, not in semantic mode. Semantically, it's just a title with two words, and then you go with css and classes to style it and split it in two lines.